pkg/config/config.go

```diff
@@ -339,6 +342,17 @@ func (c *Config) SetConfigFromEnv() error {
 		c.ImageProcessingContainerTemplate.Image = imageProcessingImage
 	}
 
+	// set environment variable for vulnerability count limit
+	vulnerabilityCountLimit := os.Getenv(VulnerabilityCountLimitEnvVar)
+	if vulnerabilityCountLimit != "" {
+		c.ImageProcessingContainerTemplate.Env = append(c.ImageProcessingContainerTemplate.Env,
+			corev1.EnvVar{
+
+				Name:  "VulnerabilityCountLimitEnvVar",
+				Value: vulnerabilityCountLimit,
+			})
+	}
```

Comment:
On the container, I think this should be an argument (which you set in image_processing.go) and not an environment variable.